www/assets/js/components/SearchFilterDrawer.tsx

```diff
@@ -72,25 +77,39 @@ export default function SearchFilterDrawer(props: Props) {
   ) : (
     <div className="controls-outer">
       <div className="controls">
-        <div onClick={openDrawer} className="filter-controls">
+        <div
+          onClick={openDrawer}
+          className="filter-controls"
+          role="button"
+          tabIndex={0}
+        >
           Filter
-          <i className="material-icons">arrow_drop_down</i>
+          <i className="material-icons" aria-hidden="true">
+            arrow_drop_down
+          </i>
         </div>
       </div>
       <div className="layout-buttons layout-buttons-mobile">
-        <button onClick={() => updateUI(null)} className="layout-button-left">
+        <button
+          onClick={() => updateUI(null)}
+          className="layout-button-left"
+          aria-label="search results with thumbnails"
+        >
           <img
             src="/images/icons/list_ui_icon.png"
             alt="search results with thumbnails"
+            aria-hidden="true"
           />
         </button>
         <button
           onClick={() => updateUI(SEARCH_COMPACT_UI)}
           className="layout-button-right"
+          aria-label="compact search results"
```

Comment:
**Suggestion:** "Show compact results" or something that conveys more of an action.